### PR TITLE
[zh-cn]Update components.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -88,15 +88,15 @@ Some types of these controllers are:
...
+  * EndpointSlice controller: Populates EndpointSlice objects (to provide a link between Services and Pods).
+  * ServiceAccount controller: Create default ServiceAccounts for new namespaces.
 -->
 这些控制器包括：
 
 * 节点控制器（Node Controller）：负责在节点出现故障时进行通知和响应
 * 任务控制器（Job Controller）：监测代表一次性任务的 Job 对象，然后创建 Pods 来运行这些任务直至完成
-* 端点控制器（Endpoints Controller）：填充端点（Endpoints）对象（即加入 Service 与 Pod）
-* 服务帐户和令牌控制器（Service Account & Token Controllers）：为新的命名空间创建默认帐户和 API 访问令牌
+* 端点分片控制器（EndpointSlice controller）：填充端点分片（EndpointSlice）对象（以提供服务和 Pod 之间的链接）。
```
Comment (howieyuen) [inline content/zh-cn/docs/concepts/overview/components.md:98]:
```suggestion
* 端点分片控制器（EndpointSlice controller）：填充端点分片（EndpointSlice）对象（以提供 Service 和 Pod 之间的链接）。
```

File: content/zh-cn/docs/concepts/overview/components.md [hunk 2]
```diff
@@ -88,15 +88,15 @@ Some types of these controllers are:
...
+* 服务账号控制器（ServiceAccount controller）：为新的命名空间创建默认的服务账号（ServiceAccounts）。
```
Comment (howieyuen) [inline content/zh-cn/docs/concepts/overview/components.md:99]:
```suggestion
* 服务账号控制器（ServiceAccount controller）：为新的命名空间创建默认的服务账号（ServiceAccount）。
```

**Review by howieyuen — COMMENTED** [review]
use keyword to indicates the name, usually in its singular form

**Comment by Sea-n** [comment]
Hey @kinzhi,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by kinzhi** [comment]
> Hey @kinzhi,
> 
> It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

Yes, I have revised it

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**